### ISSUE TITLE
allow editing Folder Path if path is missing

### ISSUE THREAD

**Issue by katrinleinweber** [syncthing/syncthing#4555]
In case of `Error on folder "…" (…): folder path missing` case `1.` from https://github.com/syncthing/syncthing/issues/4309#issuecomment-327869090 can be assumed to be true. So this FR may be functionally similar, but smaller in scope. Thus, it could serve as an experiment to gather experience, and prepare making the paths of normally syncing folders editable.

**Comment by calmh** [comment]
There is some sort of sense behind allowing path change of a stopped folder, although it's clearly still dangerous. I'm still not sure how to handle that. "Show warnings" is the developer centric solution, but we know no one ever reads warnings so it's really not a solution to the problem.

http://i.imgur.com/H0uVqFe.jpg

**Comment by bugith** [comment]
IMO if folder path is missing, the solution in no way should be in ST : user did something wrong in the FS so user will repair its FS. This way the user will understand more things... and will never blame ST has changed something in the tree. This way ST will remain a reliable/confidence program.
E.G user that shares a folder on removable drive should use fstab UUIDs or diskmgmt.msc to assign a static path to the share.
If you devs although ever decide to birth some edit path feature, please hide this into Action/Advanced .... hoho, yet it's there !!!!

**Comment by katrinleinweber** [comment]
>[...] user did something wrong in the FS so user will repair its FS.

Moving a folder on the main drive is not "wrong", yet causes the same error. Therefore, any sync software should make it easy to relocate a folder.

> [...] user will understand more things... 

Should a user understand that they have to _delete and recreate_ folder settings like [File Versioning](https://docs.syncthing.net/users/versioning.html) simply because they wanted to move a folder?

I agree that path change should not be possible accidentally, so there could be 1 click to unlock the greyed-out path before one could edit it.

**Comment by calmh** [comment]
Yeah, I think this is valid. If the folder is stopped (due to being missing or the marker being missing) the path is still greyed out, and you can click "I moved it" or something. Syncthing then says something along the lines of "You sure? If the files aren't actually the same, you're better of removing the folder and adding it again!" while allowing change of the path.

**Comment by Avamander** [comment]
I just encountered this problem and would seriously like to have a button for such folder movements, some folders are big and simply resyncing would take days, if not weeks even on a gigabit link.

**Comment by calmh** [comment]
The path is editable in the advanced config editor, it's just that there is no seatbelt or airbag installed at that point.
